Bmx.CommandLine/CommandLine.cs

```diff
@@ -18,6 +18,10 @@ class StubIdp : IIdentityProvider {
 		public Task<bool> ChallengeMfa( int selectedMfaIndex, string challengeResponse ) {
 			throw new NotImplementedException();
 		}
+
+		public Task<string[]> GetAccounts( string accountType ) {
+			throw new NotImplementedException();
```

Comment:
Double-check: this will be updated in another PR?

Reply:
For integration tests will need to write a more fleshed out mock identity provider, Think at present its best if I remove this StubIdp all together.